lib/public/utilities/remoteDataFromResponse.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RemoteData } from '/js/src/index.js';
+
+/**
+ * Parse a fetch {@see Response} into a {@see RemoteData}
+ *
+ * @param {Response} response the response to parse for remote data
+ *
+ * @return {Promise<{remoteData: RemoteData}>} the parsed remote data
+ */
+export const remoteDataFromResponse = async (response) => {
```

Comment:
This might be useful to share across all GUIs in the `Loader.js` class which implements the `fetch` method.
We can create a separate ticket for it

Reply:
Yes I agree, but it should be in a separate ticket (I improved it in another PR alongside with the modal system, that should be extracted too)